manticore/utils/log.py

```diff
@@ -11,70 +11,95 @@ def filter(self, record):
             record.stateid = '[%d]' % self.stateid
         else:
             record.stateid = ''
+
+        # Summarize logger name
+        components = record.name.split('.')
+        name = '.'.join(c[0] for c in components[:-1]) + '.' + components[-1]
```

Comment:
I think it might be worth breaking this out into a slightly more verbose function like we did with `clamp`

Reply:
👍